mdal/frmts/mdal_ply.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+ MDAL - Mesh Data Abstraction Library (MIT License)
+ Copyright (C) 2020 Peter Petrik (zilolv at gmail dot com)
```

Comment:
Probably you want to own the copyright yourself :)

Reply:
Done in the next commit